index.html

```diff
@@ -52,6 +56,147 @@ <h2 class="title2">I'm a Software Developer</h2>
+    </section>
+
+    <section class="project">
+        <div class="my-cards container">
+            <div class="card">
+                <div class="card-content">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button>See project</button>
+            </div>
+
+            <div class="card">
+                <div class="card-content">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button class="btn">See project</button>
+            </div>
+
+            <div class="card">
+                <div class="card-content">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button class="btn">See project</button>
+            </div>
+
+            <div class="card">
+                <div class="card-content">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the
+                        industry's standard.</p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <button class="btn">See project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-section">
+        <div class="about-me container">
+            <h2>About Me</h2>
+            <p>
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and
+                experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <button class="btn btn-about">Get my resume</button>
+        </div>
+
+        <hr>
+    </section>
```

Comment:
- [ ] As requested by the previous reviewer please adjust the positioning of `About-us` section's background to match the figma design

Figma design:

<img width="297" alt="Screenshot 2023-04-01 at 5 40 27 PM" src="https://user-images.githubusercontent.com/44798044/229289600-eb93c558-16d1-4e8e-8cd8-bba9e04ac2e2.png">

Your design:

<img width="357" alt="Screenshot 2023-04-01 at 5 39 26 PM" src="https://user-images.githubusercontent.com/44798044/229289579-5ce330b2-b866-4958-95a9-f52d364564cb.png">

Reply:
adjusted the positioning of About-us section's background